lib/msal-angular/docs/advanced-topics-1.x.md

```diff
@@ -0,0 +1,94 @@
+# Advanced Topics for MSAL Angular
+
```

Comment:
Recommend adding a TOC for this. Might be worth splitting this out into separate docs but I understand there may be some confusion with version #s. Could we make a separate docs folder for each version and put docs relevant for both in the top level? This way we can add a TOC in the README and leave these docs a bit smaller.

Reply:
Okay, updated. Let me know if you think it needs further changes!